### PR TITLE
Add custom payout functionality to Zergpool

### PR DESCRIPTION
**Pull request by vertiond**
This adds the ability to change the default [Zergpool](https://zergpool.com/site/faq) payout in VTC to a user-inputted BTC, LTC or DOGE address.  Pending payout information will accurately track the payout of choice although Expected Earnings (24h) still reflects an amount in VTC.

**Current Zergpool payout thresholds as of February 16, 2021:**
BTC payment threshold is 0.0025 BTC 
DOGE payment threshold is 400 DOGE and 80 DOGE on Sunday late evening(CET)
LTC payment threshold is 0.25 LTC and 0.05 LTC on Sunday late evening(CET)
*Network and exchange fees may apply

**Changelist:**
Custom payout functionality in Zergpool
Disable P2Pool in favor of centralized pools
Disable SkipVerthashVerify
Disable Testnet
Display message when "pending pool payout" info is unavailable
Avoid permission error on Windows when clicking "Save & Restart"

Further enhancement:
Expected Earnings (24h) displayed in payout of choice
Address verification

### CODE REVIEW

File: frontend/src/components/Mining.vue [hunk 1]
```diff
@@ -55,14 +55,17 @@
       </p>
       <p class="immatureBalance" v-if="balanceImmature != '0.00000000'">
         (
-        <span style="opacity: 1">{{balanceImmature}} VTC</span>
+        <span style="opacity: 1">{{balanceImmature}} {{activePayout}}</span>
```
Comment (gertjaap) [inline frontend/src/components/Mining.vue:58]: nit: i don't think we can have immature balance in other currencies, given that the pool pays us (we don't receive the foreign coin's block reward)

**Review by gertjaap — APPROVED** [review]
Looks fine to me. One nit, which i think isn't a problem anyhow.

I don think this is fundamental enough that we should put this in a v3 of OCM

File: frontend/src/components/Mining.vue (hunk 1 above)
Comment (vertiond) [reply frontend/src/components/Mining.vue]: Fixed.

**Comment by vertiond** [comment]
Closing in favor of outsourcing to a fork of OCM: https://github.com/vertiond/verthash-one-click-miner

**Pull request closed without merging**